src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -313,7 +313,11 @@ static bool getOsInfoFromFiles(nlohmann::json& info)
     {
```

Comment:
could you move line 284 to the last position of the vector?

Reply:
done